src/main/java/io/th0rgal/protectionlib/ProtectionLib.java

```diff
@@ -3,47 +3,63 @@
 import io.th0rgal.protectionlib.compatibilities.*;
 import org.bukkit.Bukkit;
 import org.bukkit.Location;
+import org.bukkit.entity.Entity;
 import org.bukkit.entity.Player;
 import org.bukkit.plugin.Plugin;
 import org.bukkit.plugin.java.JavaPlugin;
 
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Set;
 
 public class ProtectionLib {
 
-    private final static Set<ProtectionCompatibility> compatibilities = new HashSet<>();
+    private final static HashMap<CompatType,Set<ProtectionCompatibility>> compatibilities = new HashMap<>();
 
     @SuppressWarnings("Convert2MethodRef")
     public static void init(JavaPlugin plugin) {
-        handleCompatibility("WorldGuard", plugin, (m, p) -> new WorldGuardCompat(m, p));
-        handleCompatibility("Towny", plugin, (m, p) -> new TownyCompat(m, p));
-        handleCompatibility("Factions", plugin, (m, p) -> new FactionsUuidCompat(m, p));
-        handleCompatibility("Lands", plugin, (m, p) -> new LandsCompat(m, p));
-        handleCompatibility("PlotSquared", plugin, (m, p) -> new PlotSquaredCompat(m, p));
-        handleCompatibility("CrashClaim", plugin, (m, p) -> new CrashClaimCompat(m, p));
+        handleCompatibility(CompatType.PROTECTION,"WorldGuard", plugin, (m, p) -> new WorldGuardCompat(m, p));
+        handleCompatibility(CompatType.PROTECTION,"Towny", plugin, (m, p) -> new TownyCompat(m, p));
+        handleCompatibility(CompatType.PROTECTION,"Factions", plugin, (m, p) -> new FactionsUuidCompat(m, p));
+        handleCompatibility(CompatType.PROTECTION,"Lands", plugin, (m, p) -> new LandsCompat(m, p));
+        handleCompatibility(CompatType.PROTECTION,"PlotSquared", plugin, (m, p) -> new PlotSquaredCompat(m, p));
+        handleCompatibility(CompatType.PROTECTION,"CrashClaim", plugin, (m, p) -> new CrashClaimCompat(m, p));
+
+
+        handleCompatibility(CompatType.NPC,"Citizens",plugin,(m,p) -> new CitizensCompat(m,p));
```

Comment:
whats the point of the new CompatType property here?

Reply:
A server can have both a protection plugin and a npc plugin, CompatType separates them both and only loops through the npcCompat to check if the entity is an npc #isNPC()

Reply:
Also #canMine() #canPlace() etc from npc Compat will not have no-effect on protection methods because of the "CompatType" sperating the both